docs/source/Howtos/Beginner-Tutorial/Part1/Beginner-Tutorial-Building-Quickstart.md

```diff
@@ -170,74 +168,77 @@ You know you want to.
 
 ## Making Yourself a House
 
-The main command for shaping the game world is `dig`. For example, if you are standing in Limbo, you can dig a route to your new house location like this:
+The main command for shaping your game world is `dig`. For example, if you are standing in Limbo, you can dig a route to your new house location like this:
 
     dig house = large red door;door;in,to the outside;out
 
-This will create a new room named 'house'. Spaces at the start/end of names and aliases are ignored so you could put more air if you wanted.  This call will directly create an exit from your current location named 'large red door' and a corresponding exit named 'to the outside' in the house room leading back to Limbo. We also define a few aliases to those exits, so people don't have to write the full thing all the time.
+<!-- This is a confusing paragraph as related to the above command and might be more effective if it were to explicitly identify the use of commas in the arguments. -->
```

Comment:
Maybe it would be easier if one broke up the long instruting into its components, pointing out the use of `;` for aliases and the two comma-separated blocks for the exits going to the house and away from the house. Maybe even show an example without the aliases? One can also skip the return exit, but that seems to be too much info for a first taste like this ...

Reply:
I don't know what it means to "put more air" when spaces and aliases are ignored. 😕 

I'm not sure I actually understood there were comma-separated blocks until your comment here. I just knew that it was something about the semicolons and commas, but when you said "comma-separated," it clicked for me.

Reply:
The "more air" bit is saying you can do e.g  `large red door; door; in , to the outside; out` instead of `large red door;door;in,to the outside;out`.

TBH I don't think it's a very clear or necessary addition so it could be removed.

Reply:
I don't know if my revision is any better but it's certainly more concise.